### ISSUE TITLE
Fast Refresh

### ISSUE THREAD

**Issue by shuding** [nihgwu/react-runner#132]
Would be great to support HMR via React Fast Refresh. The official one was implemented as a Babel plugin so might be a bit tricky.

**Comment by nihgwu** [comment]
@shuding yes, but I think live editing is mainly target to prod env where you can't use HMR, so I think it's fine, locally I use a separate package for live editing and persist the state in url, so my edits won't get lost, actually we can use babel-standalone as well for react-runner which also provide better error message by redirecting sucrase to babel
